cloudsync/smartsync.py

```diff
@@ -362,19 +383,26 @@ def smart_info_path(self, local_path) -> Optional[SmartInfo]:
         remote_path = self.translate(REMOTE, local_path)
         if remote_path:
             ents = self.state.lookup_path(REMOTE, remote_path)
-            if ents:
+            if ents and self._include_ent(ents[0]):
                 return self._ent_to_smartinfo(ents[0], None, local_path)
         return None
 
     def smart_info_oid(self, remote_oid) -> Optional[SmartInfo]:
         ent = self.state.lookup_oid(REMOTE, remote_oid)
-        if ent:
+        if ent and self._include_ent(ent):
```

Comment:
Is there an affirmative reason not to return info for a known oid?  For example, if the file is in the middle of an active rename, then that means we can't get the info for it?

Reply:
Hmmm yeah I think the oid makes sense. I'll take this out